cmd/kubeadm/app/constants/constants.go

```diff
@@ -199,6 +199,15 @@ const (
 	// We need at least ten, because the DNS service is always at the tenth cluster clusterIP
 	MinimumAddressesInServiceSubnet = 10
 
+	// MinimumAddressesInPodSubnet defines minimum amount of pods the Pod subnet should allow.
+	// We need at least more than services, an IPv4 /28 or IPv6 /128 subnet means 14 util addressess
+	MinimumAddressesInPodSubnet = 14
+
+	// The nodes subnet mask size cannot be greater than 16 more than the pod subnet mask size
+	// xref: https://github.com/kubernetes/kubernetes/issues/44918
+	// PodSubnetNodeMaskMaxDiff limited to 16 due to the uncompressed bitmap
+	PodSubnetNodeMaskMaxDiff = 16
```

Comment:
comment above should start with PodSubnetNodeMaxDiff on the first line.

i tried to reword this, please change if needed.

> // PodSubnetNodeMaskMaxDiff is limited to 16 due to an issue with uncompressed IP bitmap in core:
> // xref: https://github.com/kubernetes/kubernetes/issues/44918
> // The node subnet mask size must be no more than pod subnet mask size + 16